esa/esa_invitations.go

```diff
@@ -0,0 +1,153 @@
+	MaxPerPage  int           `json:"max_per_page,omitempty"`
+}
+
+func (l InvitationList) String() string {
+	return Stringify(l)
+}
+
+// InvitationMember represents members that wanna send invitations
+type InvitationMember struct {
+	Member *InvitationEmails `json:"member"`
+}
+
+func (m InvitationMember) String() string {
+	return Stringify(m)
+}
+
+// InvitationEmails represents emails that wanna send invitations
+type InvitationEmails struct {
+	Emails []string `json:"emails"`
+}
+
+// GetURL fetches a team by name.
+//
+// API docs: https://docs.esa.io/posts/102#12-1-0
+func (s *InvitationsService) GetURL(ctx context.Context, team string) (*InvitationURL, *Response, error) {
+	u := fmt.Sprintf("teams/%s/invitation", team)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	url := &InvitationURL{}
+	resp, err := s.client.Do(ctx, req, url)
+	if err != nil {
+		return nil, resp, err
+	}
+	return url, resp, nil
+}
+
+// RegenerateURL regenerates an invitation URL
+//
+// API docs: https://docs.esa.io/posts/102#12-2-0
+func (s *InvitationsService) RegenerateURL(ctx context.Context, team string) (*InvitationURL, *Response, error) {
+	u := fmt.Sprintf("teams/%s/invitation_regenerator", team)
+	req, err := s.client.NewRequest("POST", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	url := &InvitationURL{}
+	resp, err := s.client.Do(ctx, req, url)
+	if err != nil {
+		return nil, resp, err
+	}
+	return url, resp, nil
+}
+
+// SendToMember send invitation emails
+//
+// API docs: https://docs.esa.io/posts/102#12-1-0
+func (s *InvitationsService) SendToMember(ctx context.Context, team string, member *InvitationMember) (*InvitationList, *Response, error) {
+	u := fmt.Sprintf("teams/%s/invitations", team)
+	req, err := s.client.NewRequest("POST", u, member)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	l := &InvitationList{}
+	resp, err := s.client.Do(ctx, req, l)
+	if err != nil {
+		return nil, resp, err
+	}
+	return l, resp, nil
+}
+
+// GetList fetches a list of invitations.
+//
+// API docs: https://docs.esa.io/posts/102#13-2-0
+func (s *InvitationsService) GetList(ctx context.Context, team string) (*InvitationList, *Response, error) {
+	u := fmt.Sprintf("teams/%s/invitations", team)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	l := &InvitationList{}
+	resp, err := s.client.Do(ctx, req, l)
+	if err != nil {
+		return nil, resp, err
+	}
+	return l, resp, nil
+}
+
+// Cancel deletes an invitation by an invitation code.
+//
+// API docs: https://docs.esa.io/posts/102#13-3-0
+func (s *InvitationsService) Cancel(ctx context.Context, team string, code string) (*Response, error) {
+	u := fmt.Sprintf("teams/%s/invitations/%s", team, code)
+	req, err := s.client.NewRequest("DELETE", u, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := s.client.Do(ctx, req, nil)
+	if err != nil {
```

Comment:
**[gosimple]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
'if err != nil { return resp, err }; return resp, nil' can be simplified to 'return resp, err' (S1013)